server/lib/allowed-features.ts

```diff
@@ -3,8 +3,30 @@ import { get } from 'lodash';
 import { types } from '../constants/collectives';
 import FEATURE from '../constants/feature';
 
+// Please refer to and update https://docs.google.com/spreadsheets/d/15ppKaZJCXBjvY7-AjjCj3w5D-4ebLQdEowynJksgDXE/edit#gid=0
 const FeatureAllowedForTypes = {
+  // 1. Contribute
+  [FEATURE.CONTRIBUTE]: [types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT],
+  [FEATURE.CONTRIBUTIONS]: [types.USER, types.ORGANIZATION, types.FUND], // unsure about these - types.COLLECTIVE, 
+  [FEATURE.RECURRING_CONTRIBUTIONS]: [types.USER, types.ORGANIZATION, types.COLLECTIVE,  types.FUND],
+  // 2. Events/Projects
+  [FEATURE.EVENTS]: [types.ORGANIZATION, types.COLLECTIVE],
+  [FEATURE.PROJECTS]: [types.FUND],
+  // 3. Budget
+  [FEATURE.BUDGET]: [types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT], // types.USER entire BUDGET section is TRANSACTIONS
+  [FEATURE.TRANSACTIONS]: [types.USER], // only USER gets entire TRANSACTIONS section
+  [FEATURE.EXPENSES]: [types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT],
+  [FEATURE.COLLECTIVE_GOALS]: [types.COLLECTIVE, types.ORGANIZATION],
+  [FEATURE.TOP_FINANCIAL_CONTRIBUTORS]: [types.COLLECTIVE, types.ORGANIZATION, types.FUND],
+  [FEATURE.ALL_FINANCIAL_CONTRIBUTORS]:[types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT],
+  // 4. Connect
+  [FEATURE.CONNECT]: [types.ORGANIZATION, types.COLLECTIVE],
   [FEATURE.CONVERSATIONS]: [types.COLLECTIVE, types.ORGANIZATION],
+  [FEATURE.UPDATES]: [types.COLLECTIVE, types.ORGANIZATION],
+  // 5. About
+  [FEATURE.ABOUT]: [types.USER, types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT],
```

Comment:
If it's allowed for all types, we don't need to set any flags here. `isFeatureAllowedForCollectiveType` will return `true` by default if there's not flag set.

```suggestion
```